examples/ex22p.cpp

```diff
@@ -160,19 +164,24 @@ int main(int argc, char *argv[])
    ComplexOperator::Convention conv =
       herm_conv ? ComplexOperator::HERMITIAN : ComplexOperator::BLOCK_SYMMETRIC;
 
-   // 3. Read the (serial) mesh from the given mesh file on all processors. We
+   // 3. Enable hardware devices such as GPUs, and programming models such as
+   //    CUDA, OCCA, RAJA and OpenMP based on command line options.
+   Device device(device_config);
+   device.Print();
```

Comment:
I just noticed this in the internal tests -- this print should be called only on rank 0.

Reply:
Thanks, @v-dobrev, for catching this. Fixed it now.